go/vt/tabletmanager/healthcheck.go

```diff
@@ -203,6 +203,8 @@ func (agent *ActionAgent) runHealthCheck(targetTabletType pbt.TabletType) {
 		if isQueryServiceRunning {
 			// we are not healthy or should not be running the
 			// query service, shut it down.
+			// Log the error which causes QueryService to stop
+			log.Errorf("About to stop query service: <%v>", err)
```

Comment:
How about "Stopping query service".
Also, it looks like we may not always have an error. We could build a 'reason' string instead based on why we set the flag, but it may be overkill.